docs/getting-started.md

```diff
@@ -88,7 +76,7 @@ volta setup && volta install ${NODE_VERSION}
 
 ## 6. Create GitHub token
 
-The GitHub token verifies team permissions. This token is necessary for the next step, [Login to Vault](#9-login-to-vault). To create a token:
+The GitHub token verifies team permissions. To create a token:
```

Comment:
I think the token is unnecessary, the only need for it that I remember was for vault access. Mine has expired and I haven't needed to regenerate one since.

Reply:
It is (rarely) used for some GitHub tasks, such as launching a GitHub Action from the command line and clearing the GitHub cache.